GitUI/CommandsDialogs/SettingsDialog/SettingsTreeViewUserControl.cs

```diff
@@ -27,6 +28,10 @@ public SettingsTreeViewUserControl()
 
             _origTextBoxFont = textBoxFind.Font;
             SetFindPrompt(true);
+
+            treeView1.ImageList = new ImageList();
+            treeView1.ImageList.ImageSize = DpiUtil.Scale(new Size(16, 16)); // Scale ImageSize and images scale automatically
```

Comment:
@drewnoakes @Quppa I'm having a bit of a conundrum. 
If I don't scale the image size I don't get `>` icons the same size as in the other treeviews.
However now I get an empty space, absent any images...

Thoughts? Suggestions?

Reply:
You could try one of the workarounds listed [here](https://stackoverflow.com/a/31507872) to hide the images.

I thought that blank space to the right of the textbox was being used to show some sort of indicator after searching - has that changed?

Reply:
>  thought that blank space to the right of the textbox was being used to show some sort of indicator after searching - has that changed?

Indeed, it is... Thanks for pointing.

Reply:
> You could try one of the workarounds listed here to hide the images.

Thank you
I don't really like the idea of custom drawing the whole thing, especially in wake of dealing with scaling issues...
[A possible solution from a mailing list](http://www.tech-archive.net/Archive/DotNet/microsoft.public.dotnet.framework.windowsforms/2006-09/msg00322.html) looked promising but it only seems to work for hovered nodes...

![image](https://user-images.githubusercontent.com/4403806/40115153-d80c469e-5952-11e8-9387-f4ebeb5e5e0b.png)

It feels like a blank image is drawn anyway.

Reply:
Any chance setting StateImageList helps (one of the other solutions listed on StackOverflow)?

Agreed about avoiding custom draw if at all possible.

Reply:
Agree that the [`StateImageList` approach](https://stackoverflow.com/a/31507872/24874) sounds promising. Did you try that one @RussKie?

Reply:
I've seen this comment, but I'm not sure how to put it to use...

Reply:
My interpretation is to replace `ImageList` with `StateImageList` and `ImageIndex` with `StateImageIndex`.

Reply:
I tried replacing `ImageList` with `StateImageList` and it doesn't work. Perhaps I'm missing something bleeding obvious....

It hurts, but I'm giving up. The proposal is not perfect, but it is more visually consistent with the rest. We can add icons for each entry later.